tests/utils/helpers.ts

```diff
@@ -127,3 +127,34 @@ export function combinePermissions(..._permissions: string[]) {
 
   return ethers.utils.hexZeroPad(result.toHexString(), 32);
 }
+
+export function encodeCompactedBytes(inputKeys: BytesLike[]) {
```

Comment:
suggesting renaming:

```suggestion
export function encodeCompactBytesArray(inputKeys: BytesLike[]) {
```